src/main/java/telraam/monitoring/BatonDetectionManager.java

```diff
@@ -60,7 +60,7 @@ public Map<Integer, List<BatonDetection>> getBatonDetections() {
             }
             var batonDetections = batonDetectionMap.get(batonId);
             var team = teamMap.get(batonTeamMap.get(batonId));
-            var batonDetection = new BatonDetection(Math.toIntExact(d.getTimestamp().getTime() / 1000), d.getRssi(), d.getStationId(), batonId, team.getName());
+            var batonDetection = new BatonDetection(Math.toIntExact(d.getTimestamp().getTime() / 1000), d.getRssi(), batonId /* FIXME: BatonDetection expects a teamId here? */, d.getStationId(), team.getName());
```

Comment:
Don't forget to make an issue ;)

Reply:
Now I have: https://github.com/12urenloop/Telraam/issues/130